From 0b9439e673180606100b6a8bcebecb5764dff3b8 Mon Sep 17 00:00:00 2001
From: Deb McLemore <debmc@linux.ibm.com>
Date: Mon, 11 Mar 2019 21:52:55 -0500
Subject: [PATCH] p9dsu: Undo slot label name changes

During some code updates the slot labels were updated to reflect
the phb layout, however expectations were that the slot labels be
aligned with the riser card slots and not the system planar slots.

References:
commit 64a16ae05bb2 ("p9dsu: Fix slot labels for p9dsu2u")
commit 87517c8737b9 ("p9dsu: Fix p9dsu slot tables")
commit 31231ed300f2 ("p9dsu: Fix p9dsu default variant")

Signed-off-by: Deb McLemore <debmc@linux.ibm.com>
---
 platforms/astbmc/p9dsu.c | 32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/platforms/astbmc/p9dsu.c b/platforms/astbmc/p9dsu.c
index d28bfca..eeb8fd7 100644
--- a/platforms/astbmc/p9dsu.c
+++ b/platforms/astbmc/p9dsu.c
@@ -100,7 +100,7 @@ static const struct slot_table_entry p9dsu1u_phb8_1_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO-R Slot2",
+		.name = "WIO-R Slot",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -120,7 +120,7 @@ static const struct slot_table_entry p9dsu1u_phb8_3_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO Slot4",
+		.name = "WIO Slot2",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -251,7 +251,7 @@ static const struct slot_table_entry p9dsu2u_phb8_1_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO-R Slot2",
+		.name = "WIO-R Slot",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -271,7 +271,7 @@ static const struct slot_table_entry p9dsu2u_phb8_3_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO Slot4",
+		.name = "WIO Slot3",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -281,7 +281,7 @@ static const struct slot_table_entry p9dsu2u_phb8_4_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO Slot5",
+		.name = "WIO Slot2",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -357,13 +357,13 @@ static const struct slot_table_entry p9dsu2uess_uio_plx_down[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0x8,0),
-		.name = "PLX SWITCH UIO Slot2",
+		.name = "PLX switch",
 		.power_limit = 75,
 	},
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0x9,0),
-		.name = "PLX DOWN LAN UIO Slot2",
+		.name = "Onboard LAN",
 	},
 	{ .etype = st_end },
 };
@@ -373,7 +373,7 @@ static const struct slot_table_entry p9dsu2uess_uio_plx_up[] = {
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
 		.children = p9dsu2uess_uio_plx_down,
-		.name = "PLX UP UIO Slot2",
+		.name = "PLX up",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -383,19 +383,19 @@ static const struct slot_table_entry p9dsu2uess_wio_plx_down[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0x1,0),
-		.name = "WIO Slot3",
+		.name = "WIO Slot1",
 		.power_limit = 75,
 	},
     	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0x8,0),
-		.name = "PLX SWITCH WIO Slot3",
+		.name = "PLX switch",
 		.power_limit = 75,
 	},
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0x9,0),
-		.name = "WIO Slot3",
+		.name = "WIO Slot2",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -406,7 +406,7 @@ static const struct slot_table_entry p9dsu2uess_wio_plx_up[] = {
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
 		.children = p9dsu2uess_wio_plx_down,
-		.name = "PLX UP WIO Slot3",
+		.name = "PLX up",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -427,7 +427,7 @@ static const struct slot_table_entry p9dsu2uess_phb0_1_slot[] = {
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
 		.children = p9dsu2uess_uio_plx_up,
-		.name = "PLX UIO Slot2",
+		.name = "PLX",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -474,7 +474,7 @@ static const struct slot_table_entry p9dsu2uess_phb8_0_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO Slot1",
+		.name = "WIO Slot3",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -484,7 +484,7 @@ static const struct slot_table_entry p9dsu2uess_phb8_1_slot[] = {
 	{
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
-		.name = "WIO-R Slot2",
+		.name = "WIO-R Slot",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
@@ -495,7 +495,7 @@ static const struct slot_table_entry p9dsu2uess_phb8_2_slot[] = {
 		.etype = st_pluggable_slot,
 		.location = ST_LOC_DEVFN(0,0),
 		.children = p9dsu2uess_wio_plx_up,
-		.name = "PLX WIO Slot3",
+		.name = "PLX",
 		.power_limit = 75,
 	},
 	{ .etype = st_end },
-- 
2.7.4

